src/main/java/org/apache/accumulo/access/AccessEvaluator.java

```diff
@@ -56,6 +55,17 @@ public interface AccessEvaluator {
    */
   boolean canAccess(AccessExpression accessExpression) throws IllegalAccessExpressionException;
 
+  /**
+   * Properly escapes an authorization string. The string can be quoted if desired.
+   *
+   * @param auth
+   * @param quote
+   * @return
+   */
+  static byte[] escape(byte[] auth, boolean quote) {
+    return AccessEvaluatorImpl.escape(auth, quote);
+  }
+
```

Comment:
I think this can be removed.  I have another comment that changes the code using this to an existing API method.

```suggestion
```

Reply:
Implemented change in 6fd8db5